### PR TITLE
fixes(insert): fix drop dispatcher when commit insert query

### PR DESCRIPTION
**Pull request by zhang2014**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

fixes(insert): fix drop dispatcher when commit insert query

## Changelog

- Bug Fix

## Related Issues

Fixes #issue

### CODE REVIEW

File: query/src/interpreters/interpreter_insert.rs [hunk 1]
```diff
@@ -160,15 +161,24 @@ impl InsertInterpreter {
...
         let append_entries = self.ctx.consume_precommit_blocks();
-        table
-            .commit_insertion(
-                self.ctx.clone(),
-                &self.plan.catalog_name,
-                append_entries,
-                self.plan.overwrite,
-            )
-            .await?;
+        let handler = self.ctx.get_storage_runtime().spawn(async move {
```
Comment (BohuTANG) [inline query/src/interpreters/interpreter_insert.rs:168]: We should add some comments to tell why we need a `spawn` here :D

**Review by dantengsky — APPROVED** [review]
LGTM

File: query/src/interpreters/interpreter_insert.rs (hunk 1 above)
Comment (zhang2014) [reply query/src/interpreters/interpreter_insert.rs:168]: Sorry, I have added and committed comment, but I forgot to push : (

Comment (BohuTANG) [reply query/src/interpreters/interpreter_insert.rs:168]: We can do it in another PR :)

**Pull request merged**